instructions/steps.md

```diff
@@ -958,14 +944,17 @@ Success!!
 Let's go over some cool things about this moment.
 
 First, we have added constraints to a method *before* implementing it.
-We talked before about how Dafny
-was ok without having a body.
-Now we show that Dafny will honor
-requirements on these stubs.
-Since Dafny will also force us
-to `ensure` these requirements in our implementation
-this kind of specification-driven development
-is a powerful tool in our toolbox.
+If you tried to compile this, Dafny would complain
+since there are no steps to execute.
+But Dafny has proved that *any* steps
+that result in these postconditions could work.
```

Comment:
```suggestion
If you tried to compile this, Dafny would complain
since there is no implementation to execute.
But Dafny has proved that *any* implementation
that results in these postconditions could work.
```

Or you could go with "body". I'd avoid "steps" both because it implies sequential statements, and you're already using Step # to program your workshop participants. :)